Gemfile

```diff
@@ -1,22 +1,24 @@
 source 'https://rubygems.org'
 
-gem 'celluloid'
-gem 'celluloid-io'
-gem 'http'
+ruby RUBY_VERSION
```

Comment:
I've been explicitly told by the Bundler developers to use `ruby RUBY_VERSION` to allow bundler to calculate variadic dependencies based on the current Ruby version, and that this behavior will become the default in future versions of Bundler.